pybamm/models/submodels/external_circuit/base_external_circuit.py

```diff
@@ -20,6 +20,8 @@ def get_fundamental_variables(self):
         variables = {
             "Discharge capacity [A.h]": Q_Ah,
             "Throughput capacity [A.h]": Qt_Ah,
+            # initial SoC is 0 as initial Q_Ah is set null
+            "Cell SoC": -Q_Ah / self.param.Q,
```

Comment:
I don't think we should define cell SoC as everyone has their own definition (see https://github.com/pybamm-team/PyBaMM/issues/2553)